app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,38 @@
+class SessionsController < ApplicationController
+
+  include LoginAuthenticator
+  before_action :find_user_by_email, only: [:create]
+  skip_before_action :authorize,     only: [:new, :create]
+
+  def new; end
+
+  def create
+    if @user.authenticate(params[:password]) && @user.activated?
```

Comment:
check if we can simplify this method, let me know if you are not sure how to do this